pkg/controlplane/reconcilers/lease_test.go

```diff
@@ -355,43 +355,21 @@ func TestLeaseEndpointReconciler(t *testing.T) {
 			endpointPorts: []corev1.EndpointPort{{Name: "baz", Port: 8080, Protocol: "TCP"}},
 			endpoints: &corev1.EndpointsList{
 				Items: []corev1.Endpoints{{
-					ObjectMeta: om("foo", true),
+					ObjectMeta: om("foo"),
 					Subsets: []corev1.EndpointSubset{{
 						Addresses: []corev1.EndpointAddress{{IP: "1.2.3.4"}},
 						Ports:     []corev1.EndpointPort{{Name: "foo", Port: 8080, Protocol: "TCP"}},
 					}},
 				}},
 			},
 			expectUpdate: &corev1.Endpoints{
-				ObjectMeta: om("foo", true),
+				ObjectMeta: om("foo"),
 				Subsets: []corev1.EndpointSubset{{
 					Addresses: []corev1.EndpointAddress{{IP: "1.2.3.4"}},
 					Ports:     []corev1.EndpointPort{{Name: "baz", Port: 8080, Protocol: "TCP"}},
 				}},
 			},
 		},
-		{
-			testName:      "existing endpoints without skip mirror label",
-			serviceName:   "foo",
-			ip:            "1.2.3.4",
-			endpointPorts: []corev1.EndpointPort{{Name: "foo", Port: 8080, Protocol: "TCP"}},
-			endpoints: &corev1.EndpointsList{
-				Items: []corev1.Endpoints{{
-					ObjectMeta: om("foo", false),
-					Subsets: []corev1.EndpointSubset{{
-						Addresses: []corev1.EndpointAddress{{IP: "1.2.3.4"}},
-						Ports:     []corev1.EndpointPort{{Name: "foo", Port: 8080, Protocol: "TCP"}},
-					}},
-				}},
-			},
```

Comment:
Removing this because the idea is that relevant Endpoints would have to already have this label at this point?

Reply:
right